SimpleUploadServer/.nvmrc

```diff
@@ -0,0 +1 @@
+v12.22.10
```

Comment:
SimpleUploadServer needs to be upgraded to the latest Apollo Server at some point but for now we have to do this. ☹️